Makefile

```diff
@@ -425,8 +425,8 @@ geoportal/c2cgeoportal_geoportal/scaffolds/create/geoportal/+package+_geoportal/
 	test -s $@
 
 geoportal/c2cgeoportal_geoportal/scaffolds/create/geoportal/+package+_geoportal/locale/en/LC_MESSAGES/+package+_geoportal-client.po:
-	mkdir --parent $(dir $@)
-	touch $@
+	$(PRERULE_CMD)
+	@echo "Nothing to be done for $@"
```

Comment:
I do not understand, seems that we need a litlle explaination on how this works

Reply:
This file don't exists because it's the source language, and he shouldn't be empty => error on msgmerge